pkg/operation/shoot/shoot.go

```diff
@@ -236,6 +237,13 @@ func (b *Builder) Build(ctx context.Context, c client.Client) (*Shoot, error) {
 		shoot.KonnectivityTunnelEnabled = konnectivityTunnelEnabled
 	}
 
+	shoot.ReversedVPNEnabled = gardenletfeatures.FeatureGate.Enabled(features.ReversedVPN)
+	if reversedVPNEnabled, err := strconv.ParseBool(shoot.Info.Annotations[v1alpha1constants.AnnotationReversedVPN]); err == nil {
```

Comment:
Can you add `AnnotationReversedVPN` also to `pkg/apis/core/v1beta1/constants/types_constants.go` and use

```suggestion
	if reversedVPNEnabled, err := strconv.ParseBool(shoot.Info.Annotations[v1beta1constants.AnnotationReversedVPN]); err == nil {
```

Reply:
Could you please share your reasoning? My understanding was that the feature is still in alpha. Therefore, we did not add it to the beta constants.
Is it the usual way to add alpha features to the beta constants as well?

Reply:
You mix different 2 things - core.gardener.cloud/v1alpha1 or core.gardener.cloud/v1beta1 are API versions and you can apply an annotation to any of them. 

> Is it the usual way to add alpha features to the beta constants as well?

Yes, for example the same is done for similar so called "alpha" annotation -> `AnnotationShootKonnectivityTunnel`.

Reply:
I am still a bit confused as I would not put an alpha feature in a beta API, but that does not matter much.
The requested change is applied in the latest commit.